packages/checkbox-input/index.js

```diff
@@ -1,45 +1,55 @@
 import React from 'react'
 import { v1 as uuidv1 } from 'uuid'
+import s from './style.module.css'
+import classNames from 'classnames'
 
-function CheckboxInput(props) {
-  //  `field` and `form` props come from Formik
+/**
+ * A basic checkbox and label that allow the user to
+ * control a `<input type="checkbox" />` element.
+ */
+function CheckboxInput({
+  label,
+  theme = { background: 'light' },
+  field,
+  form,
+  className,
+}) {
+  //  `field` and `form` props typically come from Formik,
+  // though you can also provide them without Formik.
```

Comment:
Modified this comment a bit to clarify that this component is totally usable without Formik.